methods/README.md

```diff
@@ -7,7 +7,7 @@ This is where you will define one or more [guest programs] to act as a coprocess
 
 ### Writing Guest Code
 To learn to write code for the zkVM, we recommend [Guest Code 101]. 
-Note that you don't need to write host code because the Bonsai Ethereum Relay acts as the zkVM host.
+Note that you don't need to write host code because the CLI acts as the zkVM host.
```

Comment:
It's good to explain what is acting as the host, but I imagine that many users will find this line confusing. If their never used our examples or zkVM starter template, then they will not be expecting to write host code; then won't know what host code is.